doc/rtd/topics/tutorials/qemu-debugging.rst

```diff
@@ -0,0 +1,46 @@
+.. _debug information:
+
+Qemu Tutorial Debugging
+***********************
+
+You may wish to test out the commands in this tutorial as a `script`_
+to check for copy-paste mistakes.
+
+If you successfully launched the virtual machine, but couldn't log in,
+there are a few places to check to debug your setup.
+
+To debug, answer the following questions:
+
+Did cloud-init discover the imds webserver?
```

Comment:
```suggestion
Did cloud-init discover the IMDS webserver?
```

Reply:
(You capitalised the acronym previously, but I'm not sure which is correct - please disregard this comment/suggestion if it shouldn't be capitalised)